src/SFA.DAS.LevyTransferMatching/Application/Queries/GetApplication/GetApplicationQueryHandler.cs

```diff
@@ -52,7 +52,7 @@ public async Task<GetApplicationResult> Handle(GetApplicationQuery request, Canc
                 NumberOfApprentices = application.NumberOfApprentices,
                 Sector = application.Sectors,
                 StartBy = application.StartDate,
-                TypeOfJobRole = application.StandardTitle,
+                TypeOfJobRole = application.StandardRoute,
```

Comment:
Why has this been changed? Was it incorrect before?

Reply:
I have reason to believe it was. The field was also titled TypeOfJobRole rather than Job Role.

Reply:
I have undone this change - reason being criteria matching is coming from DB precalculated